desc/compute/_geometry.py

```diff
@@ -466,6 +466,46 @@ def _perimeter_of_z(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="perimeter(z)",
+    label="P(\\zeta)",
+    units="m",
+    units_long="meters",
+    description="Perimeter of enclosed cross-section (enclosed constant zeta surface), "
```

Comment:
This computes something different than `eq.compute("perimeter(z)")` and `FourierRZToroidalSurface.compute("perimeter(z)")`. Therefore, it should have a different name.